src/assets/styles/sections/cookie-notification.scss

```diff
@@ -0,0 +1,21 @@
+.cookie-wrapper {
+    position: fixed;
+    bottom: -200px;
+    display: none;
+    z-index: 100;
```

Comment:
Do we need color and background-color at this level? We should be using theme colors. I don't want to get in a position where we're doing huge grep searches for colors the next time we change the theme. it should be matter of changing some variables values.